### PR TITLE
Adds timeframe to confirmation email

### PR DESCRIPTION
**Pull request by steverabino**
## What this does
- [x] Adds the timeframe that the user must confirm their email
  - [x] Uses Devise's config to set this, so even if it changes, the email will still be correct
  - [x] Does not run this code if Devise.confirm_within is set to nil
- [x] BONUS: Adds "Draw My Life: " to other email subject lines 

### Before
image

#### Devise.confirm_within = 7.days
image

#### Devise.confirm_within = 4.months
image

#### Devise.confirm_within = nil
image

## Deploy Instructions
No special requirements. Restart will be required if you want to change the value of Devise.confirm_within.

### CODE REVIEW

File: app/views/devise/mailer/confirmation_instructions.html.haml [hunk 1]
```diff
@@ -2,5 +2,9 @@
   Welcome #{@email}!
 %p You have been invited to contribute to the Draw My Life admin portal, collecting drawings and data to support children in crisis.
 
-%p Please click the link below to confirm your Draw My Life  account email and set your password:
+%p
+  Please click the link below
+  - unless Devise.confirm_within.nil?
+    %b within #{time_ago_in_words(Devise.confirm_within.seconds.from_now)}
```
Comment (krissy) [inline app/views/devise/mailer/confirmation_instructions.html.haml]: As we both spotted IRL just then (but relaying here for anyone else) we can use `Devise.confirm` without formatting as seconds here and it should have the same result. Also, might want to use an alternative method to "wordify" the time as `time_ago_in_words` can have some fuzzy results, e.g.

```
[6] pry(main)> helper.time_ago_in_words(Devise.confirm_within.from_now)
=> "7 days"
[7] pry(main)> Devise.confirm_within = 14.months
=> 14 months
[8] pry(main)> helper.time_ago_in_words(Devise.confirm_within.from_now)
=> "about 1 year"
[9] pry(main)> helper.time_ago_in_words(Devise.confirm_within.seconds.from_now)
=> "about 1 year"
```

**Review by krissy — COMMENTED** [review]
Thanks for making subject lines clearer too! One comment but overall approach looks good :+1:

**Comment by steverabino** [comment]
Great QA-ing! I've found that `Devise.confirm_within.inspect` will actually just print out the config as "7 days" or whatever it is set as!

image

image

**Review by krissy — APPROVED** [review]
As discussed offline, there doesn't seem to be a perfect solution to this built into ActiveSupport::Duration as per [this thread](http://stackoverflow.com/questions/15299344/rails-7-days-to-human-readable-string). This does a good job for now while the config var is set to a # of days. :+1:

**Pull request merged**